tests/vrf03.rs

```diff
@@ -278,4 +278,82 @@ mod test {
 
         Ok(())
     }
+
+    use proptest::prelude::*;
+    use vrf_dalek::constants::SEED_SIZE;
+    use vrf_dalek::errors::VrfError;
+
+    fn secret_public_keys() -> impl Strategy<Value = ([u8; SEED_SIZE], PublicKey03)> {
+        proptest::string::bytes_regex("[[:ascii:]]{32}")
+            .unwrap()
+            .prop_map(|vec| {
+                let mut bytes = [0u8; 32];
+                bytes.copy_from_slice(&vec);
+                let sk = SecretKey03::from_bytes(&bytes);
+                (bytes, PublicKey03::from(&sk))
+            })
+    }
+
+    fn payload() -> impl Strategy<Value = Vec<u8>> {
+        proptest::string::bytes_regex("[[:ascii:]]{0,254}").unwrap()
+    }
+
+    proptest! {
+        #[test]
+        fn public_key_has_32_bytes((_,pk) in secret_public_keys()) {
```

Comment:
```suggestion
        fn public_key_has_32_bytes((_sk,pk) in secret_public_keys()) {
```

lest we forget  what we are talking about :)